packages/kit/test/test.js

```diff
@@ -197,7 +197,20 @@ function duplicate(test_fn, config, is_build) {
 
 				if (start) {
 					response = await context.pages.js.goto(context.base + start);
-					await context.pages.js.evaluate(() => window.started);
+
+					// If redirecting, sometimes below error comes. Therefore wait until redirect.
+					// "page.evaluate: Execution context was destroyed, most likely because of a navigation."
+					// MOTE: waitForNavigation is extremely slow so doesn't use.
```

Comment:
```suggestion
					// NOTE: waitForNavigation is extremely slow so doesn't use.
```

Reply:
although I don't think this NOTE makes sense and I'd probably just remove it. `goto` on the previous line waits for the page `load` event, which is the same thing `waitForNavigation` would do. So it can't be any slower. Maybe the issue is that it's already been awaited for, but then the issue is waiting twice for the same event and not that the method is too slow

I think I'd just remove this line since it adds more confusion than it helps with

Reply:
Although now I'm confused because I don't know exactly where you were going to call `waitForNavigation`. The other option might be to move that comment to be right above the line where you might use it

Reply:
But In most cases, the test failed due to the bellow error without this change.

```
.svelte-kit/output/client/service-worker.js   9.25 KiB / brotli: 2.08 KiB
• • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • ✘ • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • • •   (232 / 233)

   FAIL  build:basics  "redirect-on-load [js]"
    page.evaluate: Execution context was destroyed, most likely because of a navigation.

    at Object.handler (file:///C:/Users/baseballyama/Desktop/git/svelte-contributes/kit-2424/kit/packages/kit/test/test.js:201:29)
    at async Number.runner (file:///C:/Users/baseballyama/Desktop/git/svelte-contributes/kit-2424/kit/node_modules/.pnpm/uvu@0.5.1/node_modules/uvu/dist/index.mjs:77:5)
    at async Module.exec (file:///C:/Users/baseballyama/Desktop/git/svelte-contributes/kit-2424/kit/node_modules/.pnpm/uvu@0.5.1/node_modules/uvu/dist/index.mjs:131:33)
    at async main (file:///C:/Users/baseballyama/Desktop/git/svelte-contributes/kit-2424/kit/packages/kit/test/test.js:394:2)
```

I think this is similar to https://github.com/microsoft/playwright/issues/4192.

And I can't move this waiting process to `redirect-on-load` test file.
Because this error comes before called `callback` function.

But I have modified the code in response to the comment below that you gave me.
Could you please check the updated code?